app/controllers/goldencobra/sessions_controller.rb

```diff
@@ -54,5 +50,14 @@ def logout
 
     def register
     end
+
+    private
+
+    def usermodel
+      return false if params[:usermodel].blank?
+      return false unless %w(user visitor).include?(params[:usermodel].to_s.downcase)
```

Comment:
es könnte auch in einem GoldenCobra Setting stehen und dann in der Doku beschrieben sein "wenn man sich mit einem andern Model anmelden will....", oder man prüft wie oben ob das Model z.b. irgend welchen bestimmten Kriterien entspricht... bsp: 
`MyInjectedModelName.included_modules.include?(Devise::Models::Authenticatable) => false`
aber 
`User.included_modules.include?(Devise::Models::Authenticatable) => true`